extensions/twenty/src/create-object-record.tsx

```diff
@@ -0,0 +1,117 @@
+import { Action, ActionPanel, Detail, Icon, List, showToast, Toast, useNavigation } from "@raycast/api";
+import { randomUUID } from "crypto";
+
+import twenty from "./services/TwentySDK";
+import { ObjectIcons } from "./enum/icons";
+import { OpenCreateObjectRecordForm } from "./pages";
+import { usePromise } from "@raycast/utils";
+import { useState } from "react";
+
+export default function CreateObjectRecord() {
+  const {
+    isLoading,
+    data: activeDataModels,
+    error,
+  } = usePromise(async () => {
+    const activeDataModels = await twenty.getActiveDataModels();
+
+    if (typeof activeDataModels === "string") {
+      throw new Error(activeDataModels as string);
+    }
+
+    return activeDataModels;
+  });
+
+  const [isOpenView, setIsOpenView] = useState(false);
+  const { push } = useNavigation();
+
+  if (error) {
+    return <Detail markdown={` # ERROR \n\n ${error.message}`} />;
+  }
+
+  const standardActiveModel = activeDataModels?.filter((model) => !model.isCustom);
+  const customActiveModel = activeDataModels?.filter((model) => model.isCustom);
+
+  return (
+    <List isLoading={isLoading} navigationTitle="Create Object Record" searchBarPlaceholder="Search Object Record">
```

Comment:
Minor suggestion which will give a loading indicator as well w/ no extra work:
```suggestion
    <List isLoading={isLoading || isOpenView} navigationTitle="Create Object Record" searchBarPlaceholder="Search Object Record">
```